Common/include/linear_algebra/CSysMatrix.hpp

```diff
@@ -111,7 +111,7 @@ class CSysMatrix {
   const int rank;     /*!< \brief MPI Rank. */
   const int size;     /*!< \brief MPI Size. */
 
-  enum : size_t { MAXNVAR = 8 };    /*!< \brief Maximum number of variables the matrix can handle. The static
+  enum : size_t { MAXNVAR = 20 };    /*!< \brief Maximum number of variables the matrix can handle. The static
                                                 size is needed for fast, per-thread, static memory allocation. */
```

Comment:
This was chosen arbitrarily.  NEMO, in most cases, will likely only use 11 species with a gas mixture (Can use more).